apps/web/src/components/layout/components/v2/HeaderNav.tsx

```diff
@@ -0,0 +1,210 @@
+import { ActionIcon, Avatar, Group, Header, useMantineColorScheme } from '@mantine/core';
+import * as capitalize from 'lodash.capitalize';
+import { useEffect, useMemo } from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+  colors,
+  Dropdown,
+  IconDarkMode,
+  IconHelpOutline,
+  IconLightMode,
+  IconLogout,
+  IconOutlineGroupAdd,
+  IconSettings,
+  IconTonality,
+  Text,
+  Tooltip,
+  When,
+} from '@novu/design-system';
+import { CONTEXT_PATH, IS_DOCKER_HOSTED, REACT_APP_VERSION } from '../../../../config';
+import { ROUTES } from '../../../../constants/routes.enum';
+import { useBootIntercom, useLocalThemePreference } from '../../../../hooks';
+import useTrackThemeChange from '../../../../hooks/useTrackThemeChange';
+import { discordInviteUrl } from '../../../../pages/quick-start/consts';
+import { useAuthContext } from '../../../providers/AuthProvider';
+import { useSpotlightContext } from '../../../providers/SpotlightProvider';
+import { HEADER_HEIGHT } from '../../constants';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** If this refers specifically to the height of this component, please make the names match :) ‏
```suggestion
import { HEADER_NAV_HEIGHT } from '../../constants';
```